tools/db_crashtest.py

```diff
@@ -519,6 +532,8 @@ def is_direct_io_supported(dbname):
     # tiered storage doesn't support blob db yet
     "enable_blob_files": 0,
     "use_blob_db": 0,
+    # Temperature::kCold
+    "last_level_temperature": 12,
```

Comment:
Now that we have "default_write_temperature", which is for non last levels. Maybe also set "default_write_temperature" to 4 or 8, something that is warmer than kCold, so that it's actually tiered.

Reply:
Fixed